docs/source/user-guide/general/ci_cd.rst

```diff
@@ -0,0 +1,121 @@
+=================================
+Using Deepchecks In CI/CD
+=================================
+
+CI/CD is a practice of automating steps in a software life cycle such as testing, and deploying. The same practice of
+traditional software development can be applied to machine learning, which allows for frequent and automated testing
+and validation of models, which reduce the risk of errors and improve the overall quality of the model.
+
+For example, CI/CD in machine learning can be used in scenarios such as:
+
+* Data integrity checks: Before the model training process, it’s important to validate the integrity of the data used
+  for training. This can include checks such as data completeness, missing values, and data type consistency.
+* Model training: The model is trained on the validated data set.
+* Model evaluation: The trained model is evaluated using test data and various metrics such as accuracy, precision,
+  recall, etc.
+* Model deployment: The model is deployed to production if it meets the specified criteria.
+
+
+By automating these steps through a CI/CD engine and utilizing deepchecks package in the pipeline, it ensures that the
+model is thoroughly tested and validated before it is deployed to production.
+
+Deepchecks can be used in the CI/CD process at 2 main steps of the model training process:
+
+* Before the model training process, it’s important to validate the integrity of the data used for training, and the
+  train-test split
+* After the model training process, it’s important to validate the model’s performance.
+
+In this guide we will show an end to end examples of validating both data and the trained model, but in reality
+you may want to split the process into 2 separate pipelines, one for data validation and one for model validation.
+In addition we will use the default suites which are provided by deepchecks, but you can also create your own
+custom suites which fit your needs.
```

Comment:
Instead of you, rephrase to 3rd party talking (e.g. it is recommended to split the process into 2 separate pipelines...)